src/models/Show.ts

```diff
@@ -13,6 +13,7 @@ const METADATA_VERSION: number = 1;
  * 
  * @property metadataVersion - Upon loading the show, determines what migrations are needed to make the show compatible with future CalChart versions
  * @property title           - Used to categorize saved shows
+ * @property numberDots      - How many marchers are in the show
```

Comment:
It's not necessary, but will make things a little harder on the parsing; we need a place to put the number of dots read in the .shw file.  It's better to use a scratch value, instead of in the show.  I'll remove it.